package.json

```diff
@@ -24,45 +24,45 @@
   },
   "homepage": "https://github.com/decred/dcrdata#readme",
   "devDependencies": {
-    "@babel/core": "~7.4.0",
-    "@babel/plugin-syntax-dynamic-import": "~7.2.0",
-    "@babel/preset-env": "~7.4.2",
-    "axios": "~0.18.0",
-    "babel-eslint": "~10.0.1",
-    "babel-loader": "~8.0.5",
-    "babel-polyfill": "~6.26.0",
-    "clean-webpack-plugin": "~1.0.1",
-    "css-loader": "~1.0.1",
-    "eslint": "~5.16.0",
-    "eslint-config-standard": "~12.0.0",
-    "eslint-loader": "~2.1.2",
-    "eslint-plugin-import": "^2.17.2",
-    "eslint-plugin-node": "~8.0.1",
-    "eslint-plugin-promise": "~4.1.1",
+    "@babel/core": "^7.4.5",
+    "@babel/plugin-syntax-dynamic-import": "^7.2.0",
+    "@babel/preset-env": "^7.4.5",
+    "axios": "^0.19.0",
+    "babel-eslint": "^10.0.1",
+    "babel-loader": "^8.0.6",
+    "babel-polyfill": "^6.26.0",
+    "clean-webpack-plugin": "^3.0.0",
+    "css-loader": "^2.1.1",
+    "eslint": "^5.16.0",
+    "eslint-config-standard": "^12.0.0",
+    "eslint-loader": "^2.1.2",
+    "eslint-plugin-import": "^2.17.3",
+    "eslint-plugin-node": "^9.1.0",
+    "eslint-plugin-promise": "^4.1.1",
     "eslint-plugin-react": "^7.12.4",
-    "eslint-plugin-standard": "~4.0.0",
-    "mini-css-extract-plugin": "~0.4.5",
-    "mousetrap": "~1.6.3",
+    "eslint-plugin-standard": "^4.0.0",
+    "mini-css-extract-plugin": "^0.7.0",
```

Comment:
updated mini-css-extract-plugin adds a new hot module reloading feature (not enabled).  No clue if this is of interest to us.